src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke.command;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import static duke.command.Command.*;
+
+import java.util.ArrayList;
+
+public class Duke {
+    public static void main(String[] args) {
+
+        printWelcomeMessage();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        try {
+            readFile("data/duke.txt", tasks);
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+
+        untilBye:
```

Comment:
Interesting use of label to break out of the while loop, this notation is complicated and might be confusing to some who do not understand the syntax. To keep it simple, you can indicate a flag in your `while (!isByeCommand)` then set it to `true` when we reach `case 'bye'`.

Code Quality Guideline: Maximise Readability